src/main/java/org/prebid/server/bidder/model/BidderBid.java

```diff
@@ -43,7 +39,27 @@ public static BidderBid of(Bid bid, BidType bidType, String bidCurrency) {
      */
     ExtBidPrebidVideo videoInfo;
 
+    /**
+     * Will be used by price floor enforcement. The only bidder is responsible to populate this info
+     * if bidder overrides any of request.imp[i].bidfloor or request.imp[i].bidfloorcur fields.
```

Comment:
Seems not just override, but override by price floor feature. Plus(minor) pls add dot at the end of 3 line.